buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -122,8 +170,18 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
 
         unit = "cm"
         conversion = 100.0  # cm to m TODO: Double check if this is the meaning.
-        reference_frame = "TBD"
+        reference_frame = "Arbitrary, camera"
+
+        nest_depth = len(mat_file["behavior"]["trials"]["position_trcat"])
+
+        # Merge unique descriptions if there are nested entries inside the behavior file
         description = mat_file["behavior"]["description"]
+
+        if nest_depth > 1:
+            description = ";".join(
+                np.unique(mat_file["behavior"]["description"])
+            )  # NOTE: Description is an array in this case
+
         pos_obj = Position(name=description)
```

Comment:
Also I can't comment on it since it's too deep down away the these changes; but can you include all behavior objects under the `behavior` processing module? Currently it only has the `RewardEvents` and then the position and states are outside of it (they should all be nested under `behavior`)

Reply:
Also, there's no extra reason I can see to nest this under the extra submodule called 'alternation' in the shared example file; can you store it at the same level as the linearized position?

Reply:
And similar to https://github.com/catalystneuro/buzsaki-lab-to-nwb/blob/bd7b4604443c37d80590e81c2883d83adbd70895/buzsaki_lab_to_nwb/petersen/petersenmiscdatainterface.py#L128 can you give the position modules more specific names to their constituent spatial series?